src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -961,13 +961,46 @@ public override SyntaxTree SyntaxTree
 
         internal override IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperationOptions options, CancellationToken cancellationToken)
         {
-            CSharpSyntaxNode bindableNode;
+            var operationForParent = WalkUpToBindableParentAndGetOperation(node, options, cancellationToken);
 
-            BoundNode lowestBoundNode;
-            BoundNode highestBoundNode;
-            BoundNode boundParent;
+            if (node.IsKind(SyntaxKind.Argument) && operationForParent is IHasArgumentsExpression hasArguments)
+            {
+                return TryMapArgumentNodeToOperation((ArgumentSyntax)node, hasArguments);
+            }
 
-            GetBoundNodes(node, out bindableNode, out lowestBoundNode, out highestBoundNode, out boundParent);
+            return operationForParent;
+        }
+
+        private IOperation TryMapArgumentNodeToOperation(ArgumentSyntax argumentNode, IHasArgumentsExpression hasArguments)
+        {
+            var argumentExpression = argumentNode.Expression;
+            foreach (var argument in hasArguments.ArgumentsInEvaluationOrder)
+            {
+                if (argument.Syntax == argumentExpression)
+                {
+                    return argument;
+                }
+
+                if (argument.ArgumentKind == ArgumentKind.ParamArray &&
+                    argument.Value is IArrayCreationExpression arrayCreation)
+                {
+                    foreach (var arrayElement in arrayCreation.Initializer.ElementValues)
+                    {
+                        if (arrayElement.Syntax == argumentExpression)
+                        {
+                            return argument;
+                        }
+                    }
+                }
+            }
+
+            return null;
```

Comment:
This is an tolerance path.  I'm uncertain on how i would hit it. However, i'm not comfortable with just throwing if this happens.   What is our IOperation policy here?